src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,107 @@
+import cn from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../react-app-env';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+interface Props {
+  movies: Movie[];
+  onSetMovies: (movie: Movie) => void;
+}
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ movies, onSetMovies }) => {
+  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [isMovieFound, setIsMovieFound] = useState(true);
+
+  const loadMovie = async () => {
+    const loadedMovie = await getMovie(query);
+
+    setSelectedMovie(loadedMovie);
+
+    if (selectedMovie) {
+      setIsMovieFound(true);
+    } else {
+      setIsMovieFound(false);
+    }
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={cn(
+                'input',
+                { 'is-danger': !isMovieFound },
+              )}
+              value={query}
+              onChange={(event) => {
+                setQuery(event.target.value);
+              }}
+            />
+          </div>
+
+          {!isMovieFound && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              data-cy="find"
+              onClick={() => {
+                loadMovie();
+                setQuery('');
+              }}
+            >
+              Find a movie
+            </button>
```

Comment:
Find movie previews blank movie also and its getting added also in the list after pressing add to the list button